AUTHORS

```diff
@@ -0,0 +1,5 @@
+Roman Neuhauser <rneuhauser+sumaxy@sigpipe.cz>;
```

Comment:
Good job! There is a minor issue that it's not clearly visible.
The authors list must be of the form

> :Authors:
> Author 1
> Author 2
With a leading whitespace in front of every author.

For repose I inserted that whitespace manually in front of every name in the AUTHORS file. Feel free to do the same or to come up with a better way :)